server/models/ChartsModel/index.js

```diff
@@ -1,11 +1,14 @@
 import { ObjectId } from 'mongodb'
 import { collections } from '../../utils/mongoCollections'
 
+const id = '$_id'
+const idKey = '$chart_id'
+const chartId = 'chart_id'
 const ChartsModel = {
   create: async (db, chartAttributes) =>
     await db.collection(collections.charts).insertOne(chartAttributes),
-  index: async (db, query) =>
-    await db.collection(collections.charts).find(query),
+  index: async (db, user) =>
```

Comment:
On the user model we use `all` instead of `index`, can we use it here as well for consistency?

Reply:
[e39a67e](https://github.com/AMP-SCZ/dpdash/pull/557/commits/e39a67eb3ef48faac109401996dd3d299b3fdc6b)